docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,28 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment the mistakes (coding style errors,
+neglecting commit message rules, line length and spacing etc.) on your
+Pull Request.
+
+You may be required to enhance your Pull Request by modifying your branch on
+your fork.
+
+.. important::
+
+    It is **never** allowed to close your Pull Request. **Never!**
+    It is **always** possible to modify your Pull Request using
```

Comment:
I think add a blank line above this line will help readability.

Reply:
ping

Reply:
Here, ok will do

Reply:
Still not done.  See https://github.com/coala/coala/pull/5807/files